### PR TITLE
TCM-755 | Import Facebook SDK

### PR DESCRIPTION
**Pull request by VivekV95**
### Summary

- [x] Import Facebook SDK

### Issue

[TCM-755](https://hyfnla.atlassian.net/browse/TCM-755)

### Video/Screenshot

Screen Shot 2021-07-26 at 7 20 49 PM

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -46,6 +46,7 @@ buildscript {
         amplitudeVersion = '2.30.0'
 
         stripeVersion = '16.8.0'
+        facebookVersion = '[8,9)'
```
Comment (vespertineSiren) [inline build.gradle:49]: Why is this version look so strange?
From this [link](https://github.com/facebook/facebook-android-sdk)
```
    // Facebook Android SDK (everything)
    implementation 'com.facebook.android:facebook-android-sdk:11.1.0'
```

**Review by vespertineSiren — CHANGES_REQUESTED** [review]
Just one comment about dependency on initial look through.

File: build.gradle (hunk 1 above)
Comment (VivekV95) [reply build.gradle:49]: From [here](https://developers.facebook.com/docs/android/componentsdks), but will modify to use explicit version number

Comment (VivekV95) [reply build.gradle]: [Done](https://github.com/hyfn/trx-consumer-android/pull/183/commits/752983fc66418a81092c76e822ce900dcae3a865)

**Review by onodude — APPROVED** [review]
thanks @VivekV95

**Pull request merged**